lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -520,7 +521,7 @@ def refine_tracks_gaussian(
         Fixed background parameter in photons per second.
         When supplied, the background is not estimated but fixed at this value.
     """
-    if overlap_strategy not in ("ignore", "skip", "multiple"):
+    if overlap_strategy not in ("ignore", "skip", "multiple", "simultaneous"):
```

Comment:
I think it would be good to add a deprecation warning if "multiple" is used. It's kinda weird to have 2 options with one of them being buggy, so I assume we'll want to remove this in the next major release?